app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
     <string name="app_name">Polysmee</string>
+    <string name="user_prompt">Hello user ! You are not connected. Please login !</string>
+    <string name="login">Login</string><string name="title_activity_login_check">LoginCheckActivity</string>
+    <!--
+    This string is used for square devices and overridden by hello_world in
+    values-round/strings.xml for round devices.
+    -->
+    <string name="hello_world">Hello Square World!</string>
```

Comment:
Should we have a separate xml file for each part (login/calendar/room) or one big file ?

Reply:
No i think one file is okay, it's managed by studio